site/content/en/docs/Guides/Client SDKs/cpp.md

```diff
@@ -153,3 +153,58 @@ If you wish to compile from source, you will need to compile and install the fol
 If you are building a server on Windows or macOS, and need a development build to run on
 that platform, at this time you will need to compile from source. Windows and macOS libraries
 for the C++ SDK for easier cross platform development are planned and will be provided in the near future.
+{{% /feature %}}
+{{% feature publishVersion="0.9.0" %}}
```

Comment:
@sylvainduchesne I think this is good to go, but can you give these docs a once over, and if you are good, I'll approve and merge this PR!